packages/x-components/tests/e2e/cucumber/next-queries/next-queries.spec.ts

```diff
@@ -1,5 +1,7 @@
 import { And, Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
 import { InstallXOptions } from '../../../../src/x-installer/x-installer/types';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Remove these comments please, the `@ts-ignore` is forbidden.

In order to remove the error, you need to change de `tsconfig` file inside the `tests` folder and add the rootDir to the `compilerOptions`.